src/states/DisabledState.ts

```diff
@@ -11,6 +11,12 @@ class DisabledState extends State {
     transitTo(STATE_TYPE.IDLE);
   }
 
+  public onChange(e: any, { transitTo }: FlickingContext): void {
+    // Can stop Axes's change event
+    e.stop();
+    transitTo(STATE_TYPE.IDLE);
```

Comment:
@WoodNeck 

Change event in axes occurs and if stop method is called in the move event in flicking, go to disable mode. Then, won't the change event in axes stop when it happens one more time?

Reply:
Fixed!